zstash/extract.py

```diff
@@ -388,6 +392,8 @@ def extractFiles(files, keep_files, keep_tars, multiprocess_worker=None):
             # Delete this tar if the corresponding command-line arg was used.
             if not keep_tars:
                 os.remove(tfname)
+            else:
+                os.chmod(tfname, 0o444)
```

Comment:
Would this be sufficient to make tar-files read only if we're keeping them? Or would it be better to set it based on `config.hpss == 'none'` instead of `keep_tars == True`?

Reply:
This will change default behavior even when using HPSS. I would prefer that it only applied when not using HPSS.